packages/azure-themes/src/azure/AzureStyleSettings.ts

```diff
@@ -126,6 +126,9 @@ export const AzureStyleSettings = (theme: ITheme): any => {
     DetailsRow: {
       styles: DetailsRowStyles,
     },
+    DetailsHeader: {
+      styles: DetailsHeaderStyles,
+    },
```

Comment:
There weren't explicit styles for `DetailsHeader` on the styles object so I added this only to make the checkbox inside the header always visible.

That said, the existing `DetailsListStyles` [did apply some styles to the details header via the `.ms-DetailsHeader` selector](https://github.com/microsoft/fluentui/blob/4cf4e0d300c94382390c5a9d8feaa1589f2bf641/packages/azure-themes/src/azure/styles/DetailsList.styles.ts#L73-L115). We can either move the selector styles to the `DetailsHeaderStyles` object, or consolidate the new styles with the existing selector styles.

Reply:
Smart.